### PR TITLE
VC-4227: Add msg create wallet.

### PR DESCRIPTION
**Pull request by Chuiko-GIT**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by vasylNaumenko** [comment]
LGTM

File: database/vipcoin/chain/wallets/create_wallet.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+	walletstypes "git.ooo.ua/vipcoin/chain/x/wallets/types"
+
+	"git.ooo.ua/vipcoin/lib/filter"
+
+	"github.com/forbole/bdjuno/v2/database/types"
+)
+
+// SaveCreateWallet saves the given wallets inside the database
+func (r *Repository) SaveCreateWallet(msgWallet ...*walletstypes.MsgCreateWallet) error {
```
Comment (alex-bodnar) [inline database/vipcoin/chain/wallets/create_wallet.go:16]: Может интеграционный тест написать?

File: database/vipcoin/chain/wallets/create_wallet.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+	if err := tx.Commit(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// GetCreateWallet get the given wallet from database
+func (r *Repository) GetCreateWallet(walletFilter filter.Filter) ([]*walletstypes.MsgCreateWallet, error) {
```
Comment (alex-bodnar) [inline database/vipcoin/chain/wallets/create_wallet.go:45]: Может интеграционный тест написать?

File: database/vipcoin/chain/wallets/create_wallet.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+		VALUES 
+			(:creator, :address, :account_address, :kind, :state, :extras)`
+
+	for _, wallet := range msgWallet {
+		if _, err := tx.NamedExec(query, toCreateWalletDatabase(wallet)); err != nil {
+			return err
+		}
+	}
+
+	if err := tx.Commit(); err != nil {
```
Comment (alex-bodnar) [inline database/vipcoin/chain/wallets/create_wallet.go]: return tx.Commit()

File: modules/vipcoin/chain/wallets/handle_msg.go [hunk 4]
```diff
@@ -5,15 +5,47 @@
...
 
+	switch walletMsg := msg.(type) {
+	case *typeswallets.MsgCreateWallet:
+		return m.handleMsgCreateWallet(tx, index, walletMsg)
+	default:
+		errMsg := fmt.Sprintf("unrecognized %s message type: %T", typeswallets.ModuleName, walletMsg)
+		return sdkerrors.Wrap(sdkerrors.ErrUnknownRequest, errMsg)
+	}
+
 	return nil
```
Comment (alex-bodnar) [inline modules/vipcoin/chain/wallets/handle_msg.go]: Удали, он не нужен

**Review by alex-bodnar — APPROVED** [review]

**Comment by alex-bodnar** [comment]
LGTM

**Pull request merged**